sdk/search/Azure.Search.Documents/src/Generated/Models/KnowledgeStoreProjectionSelector.cs

```diff
@@ -11,7 +11,7 @@
 namespace Azure.Search.Documents.Indexes.Models
 {
     /// <summary> Abstract class to share properties between concrete selectors. </summary>
-    public partial class KnowledgeStoreProjectionSelector
```

Comment:
This kind of change in `Azure.Search.Documents` comes from that I unified the judgement condition of abstract class in [ModelWriter](https://github.com/Azure/autorest.csharp/blob/0eec8c79d213dbabe38de654f6c9566113b3425b/src/AutoRest.CSharp/Common/Generation/Writers/ModelWriter.cs#L300) and [SerializationWriter](https://github.com/Azure/autorest.csharp/blob/0eec8c79d213dbabe38de654f6c9566113b3425b/src/AutoRest.CSharp/Common/Generation/Writers/SerializationWriter.cs#L188) of autorest.csharp.